kalite/main/api_urls.py

```diff
@@ -22,20 +19,16 @@
     url(r'^', include(ExerciseLogResource().urls)),
     url(r'^', include(AttemptLogResource().urls)),
     url(r'^', include(ContentLogResource().urls)),
-    # Retrieve video data to render a front-end video player
-    url(r'^', include(VideoResource().urls)),
     # Retrieve exercise data to render a front-end exercise
     url(r'^', include(ExerciseResource().urls)),
     # Retrieve assessment item data to render front-end Perseus Exercises
     url(r'^', include(AssessmentItemResource().urls)),
     url(r'^', include(ContentResource().urls)),
-    url(r'^exercise_log/(?P<exercise_id>\w+)$', 'exercise_log', {}, 'exercise_log'),
-    url(r'^attempt_log/(?P<exercise_id>\w+)$', 'attempt_log', {}, 'attempt_log'),
```

Comment:
Is this burned down because we don't need an API for it? Or it's been transferred to tastypie now?